### PR TITLE
 Refactoring auto configuration declarations

### PR DESCRIPTION
**Pull request by anandmnair**

### CODE REVIEW

File: rabbitmq-advanced-spring-boot-autoconfigure/src/test/java/com/societegenerale/commons/amqp/auto/configuration/SpringBootAmqpAutoConfigurationTest.java [hunk 1]
```diff
@@ -16,57 +16,28 @@
...
-        assertTrue(applicationContext.containsBean(rabbitConfig.getReQueueConfig().getExchange().getName()));
-        assertTrue(applicationContext.containsBean(rabbitConfig.getReQueueConfig().getQueue().getName()));
-      }
-    }
-
-    for(Map.Entry<String,BindingConfig> bindingEntry : rabbitConfig.getBindings().entrySet()) {
-      assertTrue(applicationContext.containsBean(bindingEntry.getKey()));
-    }
+  public void checkRabbitMqAutoConfigurationDeclaration() {
+    Mockito.verify(rabbitAdmin, Mockito.times(5)).declareExchange(Mockito.any(Exchange.class));
```
Comment (vincent-fuchs) [inline rabbitmq-advanced-spring-boot-autoconfigure/src/test/java/com/societegenerale/commons/amqp/auto/configuration/SpringBootAmqpAutoConfigurationTest.java]: don't you want to use static imports for verify and times ?

Comment (anandmnair) [reply rabbitmq-advanced-spring-boot-autoconfigure/src/test/java/com/societegenerale/commons/amqp/auto/configuration/SpringBootAmqpAutoConfigurationTest.java]: Yes, I did that just now

Comment (anandmnair) [reply rabbitmq-advanced-spring-boot-autoconfigure/src/test/java/com/societegenerale/commons/amqp/auto/configuration/SpringBootAmqpAutoConfigurationTest.java]: Thank You :)

**Review by anandmnair — COMMENTED** [review]
Included the review comments

**Review by vincent-fuchs — APPROVED** [review]

**Pull request merged**